setup.py

```diff
@@ -2,55 +2,52 @@
 
 
 install_requires = [
-    'Django>=1.8,<2.3',
-    'tqdm==4.15.0',
-    'wagtail>=1.8,<2.9',
+    "Django>=1.8,<2.3",
+    "tqdm==4.15.0",
+    "wagtail>=2.3,<2.9",
 ]
 
 
 setup_requires = [
-    'setuptools-git-version==1.0.3',
+    "setuptools-git-version==1.0.3",
 ]
 
 
 testing_extras = [
-    'coverage>=3.7.0',
-    'flake8>=2.2.0',
-    'mock>=1.0.0',
+    "coverage>=3.7.0",
+    "flake8>=2.2.0",
+    "mock>=1.0.0",
 ]
 
 setup(
-    name='wagtail-inventory',
-    url='https://github.com/cfpb/wagtail-inventory',
-    author='CFPB',
-    author_email='tech@cfpb.gov',
-    description='Lookup Wagtail pages by block content',
-    long_description=open('README.rst').read(),
-    license='CCO',
-    version='1.0',
-    version_format='{tag}.dev{commitcount}+{gitsha}',
+    name="wagtail-inventory",
+    url="https://github.com/cfpb/wagtail-inventory",
+    author="CFPB",
+    author_email="tech@cfpb.gov",
+    description="Lookup Wagtail pages by block content",
+    long_description=open("README.rst").read(),
+    license="CCO",
+    version="1.0",
+    version_format="{tag}.dev{commitcount}+{gitsha}",
     include_package_data=True,
     packages=find_packages(),
     install_requires=install_requires,
     setup_requires=setup_requires,
-    extras_require={
-        'testing': testing_extras,
-    },
+    extras_require={"testing": testing_extras,},
     classifiers=[
-        'Framework :: Django',
-        'Framework :: Django :: 1.10',
-        'Framework :: Django :: 1.11',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 2.0',
-        'Framework :: Django :: 2.1',
-        'Framework :: Django :: 2.2',
-        'Framework :: Wagtail',
-        'Framework :: Wagtail :: 1',
-        'Framework :: Wagtail :: 2',
-        'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication',
-        'License :: Public Domain',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 3',
-    ]
+        "Framework :: Django",
+        "Framework :: Django :: 1.10",
+        "Framework :: Django :: 1.11",
+        "Framework :: Django :: 1.8",
+        "Framework :: Django :: 1.9",
```

Comment:
If we're not testing against anything before 1.11, we should remove these classifiers.
```suggestion
        "Framework :: Django :: 1.11",
```

Reply:
Removed support for all versions before Django 1.11